### ISSUE TITLE
Tuple() typehint instead of Tuple[] in SimpleBulmaFinder.list

### ISSUE THREAD

**Issue by makupi** [python-discord/django-simple-bulma#51]
```
  File "C:\Users\maku\.virtualenvs\friendly-frogs-ICo_ojhS\lib\site-packages\django_simple_bulma\finders.py", line 17, in <module>
    class SimpleBulmaFinder(BaseFinder):
  File "C:\Users\maku\.virtualenvs\friendly-frogs-ICo_ojhS\lib\site-packages\django_simple_bulma\finders.py", line 171, in SimpleBulmaFinder
    def list(self, _: List[str]) -> Tuple(str, FileSystemStorage):
  File "c:\users\maku\appdata\local\programs\python\python38-32\lib\typing.py", line 724, in __call__
    raise TypeError(f"Type {self._name} cannot be instantiated; "
TypeError: Type Tuple cannot be instantiated; use tuple() instead
```

Getting this error when using `py manage.py runserver`.    
Looks like the typehint for the return type with Tuple should be `Tuple[str, FileSystemStorage]` instead of using parentheses.

**Comment by SebastiaanZ** [comment]
That's a bug, thanks.

**Comment by lemonsaurus** [comment]
Addressed in v1.3.1
